configure.ac

```diff
@@ -64,6 +64,7 @@ AC_CONFIG_FILES([Makefile
         time_interp/Makefile
         time_manager/Makefile
         constants/Makefile
+        sort/Makefile
```

Comment:
Every directory needs a Makefile.am, and every Makefile.am generates a Makefile, which must be listed in configure.ac. In this case, order is not important.